source/features/hide-follow-events-preview.js

```diff
@@ -0,0 +1,11 @@
+import select from 'select-dom';
+import {observeEl} from '../libs/utils';
+
+// Hide the preview of follow events in dashboard
+export default function () {
+	observeEl('#dashboard .news', () => {
+		for (const item of select.all('#dashboard .news .follow .body .border')) {
```

Comment:
Not sure this is the proper way of targeting the DOM element :(

Any suggestions?

Reply:
Yep! It’s correct ☺️

Reply:
I'll make the change and submit new PR tonight :)